docs/integrations/README.md

```diff
@@ -14,220 +14,221 @@ For more information about the grading system, see [Product Release Stages](http
 
 ## Sources
 
-| Connector                                                                                   | Product Release Stage| Available in Cloud? |
-|:--------------------------------------------------------------------------------------------| :------------------- | :------------------ |
-| [3PL Central](sources/tplcentral.md)                                                        | Alpha                | No                  |
-| [Adjust](sources/adjust.md)                                                                 | Alpha                | No                  |
-| [Airtable](sources/airtable.md)                                                             | Alpha                | Yes                 |
-| [AlloyDB](sources/alloydb.md)                                                               | Generally Available  | Yes                 |
-| [Amazon Ads](sources/amazon-ads.md)                                                         | Generally Available  | Yes                 |
```

Comment:
Check this change.

Reply:
Reverted and changed without yml formatting.